requirements-docs.txt

```diff
@@ -1,5 +1,6 @@
 mkdocs-material==8.5.11
 mkdocstrings==0.19.1
+mkdocstrings-python
```

Comment:
Can you check your mkdocstrings & mkdocs-material version? This should only be required with newer mkdocstrings versions, not 0.19.1 yet. See issue https://github.com/up42/up42-py/issues/357 why this is still pinned to older version.

Reply:
I think that my version is 0.19.1, but checking the docs I think the python handler is required from version 0.18.1.

![Screen Shot 2023-02-17 at 10 48 39](https://user-images.githubusercontent.com/3291933/219612827-a1641bd3-2222-4e1f-8cd3-cdc43d6ec814.png)

https://mkdocstrings.github.io/handlers/overview/

On my env it did not work (but it worked before), therefore I am not sure if they have changed something. 
Can you check if make serve work for you?